manifests/generate.sh

```diff
@@ -48,11 +48,15 @@ if [ -z ${AUTH+x} ]; then echo -e "$RED_COL Please provide auth with --auth=<aut
 if [ -z ${VERSION+x} ]; then echo -e "$RED_COL Please provide version with --version=<version> $NORMAL_COL"; exit 1; fi
 if [ -z ${PVC+x} ]; then echo -e "$RED_COL Please provide PVC with --pvc=<pvc> $NORMAL_COL"; exit 1; fi
 
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+    BASE64_ARGS="-w0"
+fi
+
 DOCKER_CONFIG=$(cat <<-END
 {
   "auths": {
     "${REGISTRY%/*}": {
-        "auth": "$(echo -n $AUTH | base64)"
+        "auth": "$(echo -n $AUTH | base64 $BASE64_ARGS)"
```

Comment:
`BASE64_ARGS ` would be undefined when run in Mac OS.

```
BASE64_ARGS: unbound variable
```

Reply:
You may use `${BASE64_ARGS:-}` instead

Reply:
fixed